webpack/scenes/ContentViews/Details/Filters/MatchContentModal/__tests__/CVRpmMatchContentModal.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { renderWithRedux, patientlyWaitFor, fireEvent } from 'react-testing-lib-wrapper';
+
+
+import api from '../../../../../../services/api';
+import CVRpmMatchContentModal from '../CVRpmMatchContentModal';
+import nock, { nockInstance, assertNockRequest, mockSetting, mockAutocomplete } from '../../../../../../test-utils/nockWrapper';
+
+const CVMatchedContent = require('./CVRpmMatchContent.fixtures.json');
+const CVMatchContentSearch = require('./CVRpmMatchContentSearch.fixtures.json');
+
+const firstMatchContent = CVMatchedContent.results[0];
+const { nvra: secondMatchContentName } = CVMatchedContent.results[1];
+const cvMatchContentPath = api.getApiUrl('/packages');
+const autocompleteUrl = '/packages/auto_complete_search';
+
+const onClose = jest.fn();
+
+const MatchContentModal =
+  <CVRpmMatchContentModal filterId={194} filterRuleId={13} onClose={onClose} />;
+
+let searchDelayScope;
+let autoSearchScope;
+beforeEach(() => {
+  searchDelayScope = mockSetting(nockInstance, 'autosearch_delay', 500);
+  autoSearchScope = mockSetting(nockInstance, 'autosearch_while_typing', true);
+});
+
+afterEach(() => {
+  nock.cleanAll();
+  assertNockRequest(searchDelayScope);
+  assertNockRequest(autoSearchScope);
+});
+
+
+test('Can show matching content modal', async (done) => {
+  const autocompleteScope = mockAutocomplete(nockInstance, autocompleteUrl);
+  const createMatchContentscope = nockInstance
```

Comment:
createMatchContentscope is a bit confusing based on other create scopes..Rename as test below.

Reply:
good call, on it